catalog-info.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: kibana
+  description: Kibana is a user interface that lets you visualize your Elasticsearch data and navigate the Elastic Stack.
+
+  annotations:
+    backstage.io/source-location: url:https://github.com/elastic/kibana/tree/main
+    github.com/project-slug: elastic/kibana
+    github.com/team-slug: elastic/kibana-tech-leads
+    buildkite.com/project-slug: elastic/kibana
+
+  tags:
+    - typescript
+    - javascript
+    - dashboards
+    - metrics
+    - visualizations
+    - observability
+
+  links:
+    - title: Documentation
+      url: https://www.elastic.co/guide/en/kibana/current/index.html
+
+spec:
+  type: service
+  owner: group:kibana-tech-leads
```

Comment:
I'm not sure what we use the `owner` field for in Backstage, but I'm thinking if this should be either `kibana-tech-leads` as you put down here, or instead `kibana-operations`?

Reply:
It has just an informational character to enrich teams and user informations, nothing else. 
Ideally, I think here the owner should be actually everyone who contributes to it.
But most likely there is no group like this. Tech-leads is maybe fine.

Reply:
Cool. Let's keep it like that for now. We can always change it if we feel there's a better group

Reply:
Yeah, I wish there was space for multiple listings here.  I can swap this to kibana operations.